### PR TITLE
:bug: time to read bug

### PR DESCRIPTION
**Pull request by avi312singh**

### CODE REVIEW

File: src/components/cards/card.jsx [hunk 1]
```diff
@@ -54,6 +55,12 @@ Card.propTypes = {
             timeToRead: PropTypes.number,
         }).isRequired,
     }).isRequired,
+    body: PropTypes.shape({
+        childMarkdownRemark: PropTypes.shape({
+            html: PropTypes.string,
```
Comment (andgreendigital) [inline src/components/cards/card.jsx]:
```
html: PropTypes.string,
```
We don't require this just for time to read!

File: src/templates/__snapshots__/blog-post.test.jsx.snap [hunk 2]
```diff
@@ -45,7 +45,6 @@ exports[`<BlogPost/> render with default/required props 1`] = `
         }
       }
     >
-      0
```
Comment (andgreendigital) [inline src/templates/__snapshots__/blog-post.test.jsx.snap]: These snapshots should display test data

**Comment by eugene-matvejev** [comment]
merge

File: src/components/cards/card.jsx (hunk 1 above)
Comment (avi312singh) [reply src/components/cards/card.jsx]: Not sure, everything is required imo

File: src/templates/page-template.jsx [hunk 3]
```diff
@@ -140,10 +145,15 @@ query componentsForHomePage($slug: String!) {
                             }
                             description {
                                 childMarkdownRemark {
-                                    timeToRead
                                     html
                                 }
                             }
+                            body {
+                              childMarkdownRemark {
```
Comment (eugene-matvejev) [inline src/templates/page-template.jsx]: where we'll be using body.html ?

File: src/components/cards/card.test.jsx [hunk 4]
```diff
@@ -24,6 +24,29 @@ describe('<Card/>', () => {
             },
         },
         description: {
+            childMarkdownRemark: {
+                excerptAst: {
```
Comment (eugene-matvejev) [inline src/components/cards/card.test.jsx]: `html: '{{html}}'` is enough

**Comment by andgreendigital** [comment]
Please resolve conflicts and make sure there are no lint errors!

File: src/templates/page-template.jsx [hunk 5]
```diff
@@ -205,11 +214,18 @@ const resolve = ({ __typename, ...props }, i) => {
...
+            />
+        );
     case 'ContentfulCardContainer':
-        if (props.gradientHexCodeTopRight && props.gradientHexCodeBottomLeft) {
-            style = { background: `linear-gradient(to bottom left, ${props.gradientHexCodeTopRight}, ${props.gradientHexCodeBottomLeft})` };
-        }
+        const style = props.gradientHexCodeTopRight && props.gradientHexCodeBottomLeft
+            ? { background: `linear-gradient(to bottom left, ${props.gradientHexCodeTopRight}, ${props.gradientHexCodeBottomLeft})` }
+            : { };
+
```
Comment (andgreendigital) [inline src/templates/page-template.jsx]: Why have these changes been made?

Comment (eugene-matvejev) [reply src/templates/page-template.jsx]: you don't initiate empty object if it is not needed :)

**Pull request merged**